include/dev/MAX31855.hpp

```diff
@@ -22,12 +23,32 @@ class MAX31855 {
      */
     explicit MAX31855(SPI& spi, uint8_t device);
 
+    /*
```

Comment:
We want this to be picked up by Doxygen, so it should have two asterisks. Can you see if there's a standard way of documenting enums in Doxygen?
```suggestion
    /**
```